templates/nginx_stage.yml.erb

```diff
@@ -28,7 +28,9 @@ ondemand_portal: '<%= scope['openondemand::nginx_stage_ondemand_portal'] -%>'
 # NB: If this is not set then most apps will use default title "Open OnDemand"
 #
 #ondemand_title: null
+<% if scope['openondemand::nginx_stage_ondemand_title'] != nil -%>
```

Comment:
```suggestion
<% if scope['openondemand::nginx_stage_ondemand_title'] -%>
```

Reply:
This is the convention used else where in this template.

Reply:
Fixed